R/dash.R

```diff
@@ -45,9 +45,13 @@ Dash <- R6::R6Class(
     #' @param requests_pathname_prefix Character. A prefix applied to request endpoints
     #' made by Dash's front-end. Environment variable is `DASH_REQUESTS_PATHNAME_PREFIX`.
     #' @param external_scripts List. An optional list of valid URLs from which
-    #' to serve JavaScript source for rendered pages.
+    #' to serve JavaScript source for rendered pages. Each entry can be a string (the URL)
+    #' or a list with `src` (the URL) and optionally other `<script>` tag attributes such
```

Comment:
```suggestion
    #' or a named list with `src` (the URL) and optionally other `<script>` tag attributes such
```
It might be worth specifying named list in this case, if an unnamed list is an invalid input.